Makefile.unx

```diff
@@ -44,10 +44,16 @@ endif
 default: lib
 include $(foreach unit,$(units), $(unit)/Makefile.am)
 
+ifeq ($(arch),aarch64)
+include aes/aarch64/Makefile.am
```

Comment:
This is getting messy.  Why not just add `units += aes` to this if block and the last change with vpath will be unnecessary?

Reply:
XTS and CBC are not implemented. `units += aes ` will build XTS and CBC tests , that will break the build .

I will change it to `units += aes` after XTS and CBC tests done.